src/discounts/index.tsx

```diff
@@ -63,7 +63,7 @@ const SaleCreateView: React.FC<RouteComponentProps> = ({ location }) => {
 };
 
 const VoucherListView: React.FC<RouteComponentProps<{}>> = ({ location }) => {
-  const qs = parseQs(location.search.substr(1));
+  const qs = parseQs(location.search.substr(1), { arrayLimit: Infinity });
```

Comment:
Document why you need to do it this way, it looks confusing. Doesn't it also touch other parts of the app?